core/src/script/CGXP/plugins/MyPosition.js

```diff
@@ -70,19 +95,31 @@ cgxp.plugins.MyPosition = Ext.extend(gxp.plugins.Tool, {
         if (!('geolocation' in navigator)) return [];
 
         var map = this.target.mapPanel.map;
+        var layer = new OpenLayers.Layer.Vector("Geolocation");
```

Comment:
```
* @include OpenLayers/Layer/Vector.js
* @include OpenLayers/Feature/Vector.js
```
must be added in the include/requires block at the beginning of the file (maybe also for OpenLayers.Geometry.Point and OpenLayers.Geometry.Polygon, it doesn't hurt to add them).